pkg/receiver/activedirectoryinvreceiver/factory.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package activedirectoryinvreceiver
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver"
+)
+
+const (
+	// The value of "type" key in configuration.
+	typeStr = "activedirectoryinv"
```

Comment:
There's an upstream [Active Directory Domain Services receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.88.0/receiver/activedirectorydsreceiver/README.md) that has type name `active_directory_ds`. Perhaps we should also use underscores in our receiver's name: `active_directory_inv`? Or `active_directory_inventory`?

Reply:
I'll go with `active_directory_inventory`